week-2/to_do_list/to_do_list.rb

```diff
@@ -0,0 +1,48 @@
+require_relative "to_do_class"
+
+def operation(user_input, task_list)
+  #LIAN y DIEGO
+  #Ejecuta acción según el input del usuario
+
+  if !!(user_input =~ /^t$/) 
```

Comment:
This is a little bit confused. Maybe you can use the `scan` method ? 
Something like user_input.scan(/^t$/).any?